properties.schema

```diff
@@ -111,21 +124,140 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Defines the Item text / icon colour when selected. Should be a colour that provides good contrast against the Item selected colour."
+            "help": "Defines the item text / icon colour when selected. Should be a colour that provides good contrast against the item selected colour."
           },
           "visited": {
             "title": "Visited colour",
```

Comment:
As this variable is only used by presentation components I would change the title for consistency with the other component item colours.
_Item colour - visited_

Reply:
_help_ text will need updating too.

Reply:
This should now be resolved